### PR TITLE
Michael landers

### PR DESCRIPTION
**Pull request by mlanders**

### CODE REVIEW

File: src/room.py [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+from player import Player
```
Comment (jeffyjkang) [inline src/room.py:1]: room class

File: src/player.py [hunk 2]
```diff
@@ -1,2 +1,36 @@
 # Write a class to hold player information, e.g. what room they are in
```
Comment (jeffyjkang) [inline src/player.py:1]: player class

File: src/item.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Item:
```
Comment (jeffyjkang) [inline src/item.py:1]: item class

File: src/adv.py [hunk 4]
```diff
@@ -51,48 +51,65 @@
 print(f'Welcome {player.name} \n_________________________________________')
```
Comment (jeffyjkang) [inline src/adv.py]: movement

File: src/adv.py [hunk 5]
```diff
@@ -36,66 +36,66 @@
 room['narrow'].n_to = room['treasure']
```
Comment (jeffyjkang) [inline src/adv.py:37]: pickup

**Review by jeffyjkang — APPROVED** [review]
Hey Michael, great stuff on the MVP, it can be a little hard to navigate, some redundant commands, not sure if it was on purpose, and i cant figure out how the inventory check works, but i like the added item weights and added details, and overall great work!!!

File: src/room.py (hunk 1 above)
Comment (mlanders) [reply src/room.py]: for checking you can go into the action and write "check inventory" or "check room"

**Comment by mlanders** [comment]
for checking you can go into the action and write "check inventory" or "check room"

**Pull request merged**